Jenkinsfile

```diff
@@ -2,19 +2,41 @@ pipeline {
     agent any
 
     stages {
-        stage('Build') {
-            steps {
-                echo 'Building..'
-            }
-        }
         stage('Test') {
-            steps {
-                echo 'Testing..'
+            parallel {
+                stage('Python3.6') {
+                    steps {
+                        withPythonEnv('CPython-3.6') {
+                            sh '''
+                                pip3 install numpy==1.18.* wheel
+                                pip3 install -e .
+                                export MPLBACKEND="agg"
+                                pytest -v
+                            '''
+                        }
+                    }
+                }
+                stage('Python3.7') {
+                    steps {
+                        withPythonEnv('CPython-3.7') {
+                            sh '''
+                                pip3 install numpy==1.18.* wheel
+                                pip3 install -e .
```

Comment:
Pourquoi python setup.py install does not work ?

Reply:
It work but in the end it do the same thing and I use more this command than setup.py